builder/vagrant/driver_2_2.go

```diff
@@ -199,7 +199,11 @@ func (d *Vagrant_2_2_Driver) vagrantCmd(args ...string) (string, string, error)
 	cmd.Stdout = &stdout
 	cmd.Stderr = &stderr
 
-	cmd.Start()
+	err := cmd.Start()
+	if err != nil {
+		return "", "", fmt.Errorf("Error starting vagrant command with args: %s",
```

Comment:
I recently found out about the `%q` option of fmt and it's awesome at quoting strings for us : https://play.golang.org/p/x2R2sNFZyFa

```suggestion
		return "", "", fmt.Errorf("Error starting vagrant command with args: %q",
```

Reply:
Also; nice a new error is handled !!! 💯